handlers/imprint.go

```diff
@@ -11,9 +11,10 @@ import (
 
 // Imprint will show the imprint data using a go template
 func Imprint(w http.ResponseWriter, r *http.Request) {
+	templateFiles := []string{getImprintPath()}
```

Comment:
I don't understand your change here.

Let's say there is an imprint template configured and the template exists. Now,
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/30915504f9deaf93057c6bd66e645e95ddde89c3/handlers/imprint.go#L14
would add something (EXCLUDING the layout templates) and therefore
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/30915504f9deaf93057c6bd66e645e95ddde89c3/handlers/imprint.go#L15
would fail and the layout is not added.

Navigate to https://beaconcha.in/imprint, it look like you should ALWAYS add the layout templates here.

Reply:
As discussed, this shooould work.